llama_index/embeddings/voyageai.py

```diff
@@ -0,0 +1,59 @@
+"""OpenAI embeddings file."""
+
+from typing import List, Optional
+
+import voyageai
+
+from llama_index.embeddings.base import BaseEmbedding
+
+
+class VoyageEmbedding(BaseEmbedding):
+    """Class for Voyage embeddings.
+
+    Args:
+        model (str): Model for embedding.
+            Defaults to "voyage-01".
+
+        deployment_name (Optional[str]): Optional deployment of model. Defaults to None.
+            If this value is not None, mode and model will be ignored.
+            Only available for using AzureOpenAI.
+    """
+
+    model: str = "voyage-01"
+    voyage_api_key: Optional[str] = None
+
+    def __init__(
+        self,
+        model: str = "voyage-01",
+        voyage_api_key: Optional[str] = None,
+    ):
+        voyageai.api_key = voyage_api_key
```

Comment:
Let's please add the import of voyageai library here. This follows a similar pattern for other embedding classes (see for example [here](https://github.com/run-llama/llama_index/blob/14a059f6318c0b30c7799ac6f882d372aaf6a1d4/llama_index/embeddings/clarifai.py#L41)) as well as other integrations throughout the library:

```python
        try:
            import voyageai
        except ImportError:
            raise ImportError(
                "voyageai package not found, install with" "'pip install voyageai'"
            )
```

In doing this way we can remove `voyageai` as a dependency in the overall library (i.e. remove entry from `pyproject.toml`).

Reply:
It looks like you place this statement at the top of this file. Please move it here at L.39 instead. That way, the import only happens if user is trying to create an object from this class.